pkgs/games/rbdoom-3-bfg/default.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, directx-shader-compiler
+, libGLU
+, openal
+, SDL2
+, vulkan-headers
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rbdoom-3-bfg";
+  version = "1.4.0";
+  src = fetchFromGitHub {
+    owner = "RobertBeckebans";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-r/dvTirgFXdBJ+Gjl6zpHoGCTPoo0tRmOCV9oCdnltI=";
+    fetchSubmodules = true;
+  };
+  nativeBuildInputs = [
+    cmake
+    directx-shader-compiler
+    libGLU
+    openal
+    SDL2
+    vulkan-headers
+  ];
+
+  configurePhase = ''
+    mkdir build
+    cd build
+    cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Release -DFFMPEG=OFF -DBINKDEC=ON ../neo
+  '';
```

```suggestion
  cmakeDir = "../neo";
  cmakeFlags = [
    "-DFFMPEG=OFF"
    "-DBINKDEC=ON"
  ];
```

Let the CMake setup hooks handle this part for you, if you can :)

---

Oh yeah, and also see if you can enable `USE_SYSTEM_ZLIB` and stuff too, to avoid using their vendored versions

---

thank you, that does look better. both fixed.